docs/ruleset_functions/get_energy_required_to_heat_swh_use.md

```diff
@@ -0,0 +1,67 @@
+
+## get_energy_required_to_heat_swh_use
+
+Description: This function calculates the total energy required to heat the SWH use over the course of a year in btu
+
+Inputs:
+- **swh_use**
+- **space**
+
+Returns:
+- **energy_required**: A fixnum indicating the total btu required to heat the water indicated by the swh_use
+
+Function Call:
+
+- 
+
+Data Lookup: None
+
+Logic:
+
+- get the schedule: `hourly_schedule = swh_use.use_multiplier_schedule`
+- get the temperature at fixture: `temp_at_fixture = shw_use.temperature_at_fixture`
+- get the distribution system: `distribution_system = shw_use.served_by_distribution_system`
+- get the water used: `swh_use_value = swh_use.use`
+- get the inlet water temperature: `inlet_T_sched = distribution.entering_water_mains_temperature_schedule`
+- set the drain heat recovery efficiency to 0: `drain_heat_recovery_efficiency = 0`
+- get the drain heat recovery efficiency if heat is recovered by drain: `if(swh_use.is_heat_recovered_by_drain): drain_heat_recovery_efficiency = distribution.drain_heat_recovery_efficiency`
+- get the use_units: `use_units = swh_use.use_units`
+- convert the shw_water_used from whatever units it's currently in to volume (assumed to be gallons)
```

Comment:
This does not appear to be what these are doing which is fine but I think this labeling is incorrect. The next lines are not converting to volume if the units are not related to volume.

Reply:
updated the comment to reflect what's happening in the logic